Makefile

```diff
@@ -87,6 +89,7 @@ install-doc-gen: apply-doc-gen-build
 ## Update OpenShift resources related to the Document Generation image.
 apply-doc-gen-build:
 	cd ods-doc-gen-svc/ocp-config && tailor apply --namespace ${NAMESPACE}
+	ocp-scripts/import-image-from-dockerhub.sh --namespace ${NAMESPACE} --imagename ods-document-generation-svc --targetimagestream ods-doc-gen-svc
```

Comment:
Should we just rename the image stream? Since we move to `ods` namespace anyway ...